src/OrchardCore.Modules/OrchardCore.PublishLater/Indexes/PublishLaterPartIndex.cs

```diff
@@ -7,7 +7,7 @@ namespace OrchardCore.PublishLater.Indexes
 {
     public class PublishLaterPartIndex : MapIndex
     {
-        public DateTime? ScheduledPublishUtc { get; set; }
+        public DateTime? ScheduledPublishUtcDateTime { get; set; }
```

Comment:
```suggestion
        public DateTime? ScheduledPublishDateTimeUtc { get; set; }
```

Maybe? just to keep the consistency with Utc at the end, is often useful

Reply:
I would have gone with ScheduledPublishUtc just like we do with CreatedUtc. Not sure there is a need for repetition of the dbtype in the name.